### PR TITLE
[s]No more mindswapping with bosses

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
I thought of lazarus injectors, I thought of sentience potions, I thought of trasferrence potions, but somehow I missed the most obvious one

### CODE REVIEW

**Comment by coiax** [comment]
Are you sure you want to prevent a wizard purposefully going out and finding a dragon and getting next to it, not dying and then controlling its body? Sounds kind of badass tbh.

**Comment by KorPhaeron** [comment]
It'll be cool like, once, until every wizard goes and does that (or even worse, gets Legion)

File: code/modules/spells/spell_types/mind_transfer.dm [hunk 1]
```diff
@@ -35,6 +35,10 @@ Also, you never added distance checking after target is selected. I've went ahea
 		user << "<span class='warning'>They are too far away!</span>"
 		return
 
+	if(istype(target, /mob/living/simple_animal/hostile/megafauna))
+		user << "<span class='warning'>This creature is too powerful to control!</span>"
+		return
```
Comment (KazeEspada) [inline code/modules/spells/spell_types/mind_transfer.dm]: Nice snowflake TBH

**Comment by KorPhaeron** [comment]
@Cheridan @ChangelingRain 

Please merge before someone mindswaps into the cheriskull

**Comment by PKPenguin321** [comment]
:-1: What coiax said
Getting to and surviving on lavaland would require a wiz hardsuit and extraordinary effort
If it ever actually happens that would be badass

**Comment by KorPhaeron** [comment]
>Getting to and surviving on lavaland would require a wiz hardsuit

That's not true though

**Comment by coiax** [comment]
Kor's right, lavaland just needs internals.

**Comment by Zilenan91** [comment]
Did this actually happen in a round

**Pull request merged**